specification/sdk-tracing.md

```diff
@@ -260,6 +260,53 @@ high contention in a very high traffic service.
 * `maxExportBatchSize` - the maximum batch size of every export. It must be
   smaller or equal to `maxQueueSize`. The default value is `512`.
 
+##### Span watcher processor
+
+This implementation passes spans that have been started but not yet ended
+to the configured exporter.
+On each interval defined by `reportIntervalMillis`, all un-ended spans that
+have been started more than `reportIntervalMillis` ago, are exported as is.
+
+**Configurable parameters:**
+
+* `exporter` - the exporter where the spans are pushed.
+* `maxWatchedSpans` - the maximum number of un-ended spans to be watched.
```

Comment:
The term "un-ended" feels really cumbersome to me. Let's add some initial terminology clarification -  e.g:

> Such spans that have been started but not yet ended will be referred to as "processing|open|... spans" from now on.

Reply:
I replaced it with "unfinished" and defined the terminology in the introduction. What do you think? _Open_ would be an option but I'd rather only use that if the start/end methods were called open and close.